dll/win32/shell32/wine/classes.c

```diff
@@ -67,14 +94,6 @@ BOOL HCR_MapTypeToValueW(LPCWSTR szExtension, LPWSTR szFileType, LONG len, BOOL
 	  return FALSE;
 	}
 
-#ifdef __REACTOS__
-        if (!RegLoadMUIStringW(hkey, L"FriendlyTypeName", szFileType, len, NULL, 0, NULL))
```

Comment:
What happens to this "FriendlyTypeName" now? Was this handling actually unneeded/unwanted?

Reply:
Yes, it was incorrect.